pkgs/by-name/mu/multimidicast/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, alsa-lib }:
+stdenv.mkDerivation rec {
+  name = "multimidicast";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://llg.cubic.org/tools/multimidicast/multimidicast-${version}.tar.gz";
+    hash = "sha256-NmDPQS+Rib1ro20PrhDYHX7kAvhmENlPKSptljA8vh4=";
+  };
+
+  nativeBuildInputs = [ alsa-lib ];
```

```suggestion
  buildInputs = [ alsa-lib ];
```
libraries go to `buildInputs`, executables (used at build time) go to `nativeBuildInputs`